src/pudl/settings.py

```diff
@@ -571,6 +584,7 @@ class DatasetsSettings(FrozenBaseModel):
     phmsagas: PhmsaGasSettings | None = None
     nrelatb: NrelAtbSettings | None = None
     gridpathratoolkit: GridPathRAToolkitSettings | None = None
+    vcerare: VCERareSettings | None = None
```

Comment:
another nit, but I think even the VCE can be lowercased to VceRare... to match the others.